packages/core/src/domain/tracekit.ts

```diff
@@ -931,12 +931,16 @@ export const computeStackTrace = (function computeStackTraceWrapper() {
     }
 
     return {
-      message: ex.message,
-      name: ex.name,
+      message: forceString((ex as any).message),
+      name: forceString((ex as any).name),
       stack: [],
     }
   }
 
+  function forceString(candidate: unknown) {
+    return typeof candidate !== 'string' ? JSON.stringify(candidate) : candidate
```

Comment:
In this PR, I would do only `candidate === undefined ? undefined : String(candidate)`. `JSON.stringify` can throw, for example if `candidate` is an `Element` (related issue: https://github.com/DataDog/browser-sdk/issues/568) .

Reply:
Not sure to get your point, FMU in the linked issue, the case is about DOM event handling not about an exception due to `JSON.stringify` of an `Element` 😕 
`JSON.stringify` is already what we are using in similar cases without message property, cf [formatUnknownError](https://github.com/DataDog/browser-sdk/blob/master/packages/core/src/tools/error.ts#L31), any reason to have a different behavior here?

Reply:
My point is: `JSON.stringify` can throw. I think we should use something safer like `jsonStringify` or `String`.

Reply:
OK, I get it.
not needed with the updated implementation